code/modules/admin/verbs/adminhelp.dm

```diff
@@ -387,7 +388,7 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 	if(state != AHELP_ACTIVE)
 		return
 
-	var/msg = "<span class ='adminhelp'>Your ticket is now being handled by an admin. Please be patient.</span>"
+	var/msg = "<span class ='adminhelp'>Your ticket is now being handled by an admin. Please wait while they type their response and gather relevant information.</span>"

```

Comment:
"Please be patient" is a lot more to-the-point and doesn't imply that the admin is going to be immediately typing up a response to an ahelp. The only thing this will do is make players more prone to being impatient when admins smack that "handle" button

Reply:
Justin, one of your better admins, requested this feature by the way. They think it's useful so who am I to argue?